RadixWallet/Core/SharedModels/Entities/EntitiesControlledByFactorSource.swift

```diff
@@ -56,23 +56,23 @@ extension EntitiesControlledByFactorSource {
 	}
 
 	/// Non hidden
-	public var babylonAccounts: NonEmptyAccounts? {
-		NonEmpty(rawValue: accounts.filter(not(\.isOlympiaAccount)).asIdentifiable())
+	public var babylonAccounts: IdentifiedArrayOf<Profile.Network.Account> {
+		accounts.filter(not(\.isOlympiaAccount)).asIdentifiable()
 	}
 
 	/// hidden
-	public var babylonAccountsHidden: NonEmptyAccounts? {
-		NonEmpty(rawValue: hiddenAccounts.filter(not(\.isOlympiaAccount)).asIdentifiable())
+	public var babylonAccountsHidden: IdentifiedArrayOf<Profile.Network.Account> {
+		hiddenAccounts.filter(not(\.isOlympiaAccount)).asIdentifiable()
 	}
 
 	/// Non hidden
-	public var olympiaAccounts: NonEmptyAccounts? {
-		NonEmpty(rawValue: accounts.filter(\.isOlympiaAccount).asIdentifiable())
+	public var olympiaAccounts: IdentifiedArrayOf<Profile.Network.Account> {
```

Comment:
Do we ever query this by id? Even if we do, O(n) vs O(1) shouldn't matter very much since you are unlikely to have tens of thousands of accounts. Maybe there are other reasons, like enforcing uniqueness?